cli/analytics.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/bitrise-io/bitrise-plugins-analytics/analytics"
+	"github.com/bitrise-io/bitrise-plugins-analytics/configs"
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/bitrise/plugins"
+	log "github.com/bitrise-io/go-utils/log"
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+)
+
+func ensureFormatVersion(pluginFormatVersionStr, hostBitriseFormatVersionStr string) (string, error) {
+	if hostBitriseFormatVersionStr == "" {
+		return "This analytics plugin version would need bitrise-cli version >= 1.6.0 to submit analytics", nil
```

Comment:
makes no sense to write exact version in the log, as it is always checning which version of the CLI is required

Reply:
actually this version is not changing, as this is the minimum version for every plugin, we introduced plugin handling in this version